dao/src/test/java/com/iluwatar/dao/DbCustomerDaoTest.java

```diff
@@ -184,20 +181,21 @@ public void updationShouldBeSuccessAndAccessingTheSameCustomerShouldReturnUpdate
    * DB service unavailable.
    * 
    */
+  @Nested
   public class ConnectivityIssue {
     
     private static final String EXCEPTION_CAUSE = "Connection not available";
-    @Rule public ExpectedException exception = ExpectedException.none();
+    //@Rule public ExpectedException exception = ExpectedException.none();
     
     /**
      * setup a connection failure scenario.
      * @throws SQLException if any error occurs.
      */
-    @Before
+    @BeforeEach
     public void setUp() throws SQLException {
       dao = new DbCustomerDao(mockedDatasource());
-      exception.expect(Exception.class);
-      exception.expectMessage(EXCEPTION_CAUSE);
+      //exception.expect(Exception.class);
+     //exception.expectMessage(EXCEPTION_CAUSE);
```

Comment:
Remove commented code

Reply:
Sorry for this leftover, let me submit separate PR